New york times

biodunJohnson opened the pull request:
need a check up on my page

uryelah commented:
Hello **James Omage**, I'm Sarah, your reviewer for this PR 👩‍🚀  

**First of all great job with this project the site looks almost identical to the original but a few differences like the search icon.🌟**

There are still a couple of issues to fix before moving on so read the requirements below carefully and keep your code game strong 💪 

---------     

## Required Changes 🏗 👷‍♀ 

- [x]  Please make a more descriptive Pull Request message listing what you accomplished in this project. Check [the link ](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/)and gif below for reference:

![pr_message](https://user-images.githubusercontent.com/16193212/80316132-712edd00-87d2-11ea-888f-2a4750b05dbf.gif)


- [x] Make sure there are no errors or warnings in your[ HTML validation.](https://validator.w3.org/nu/)

- [x] From now on remember to capitalize your commit messages.

----

**Kindly fix the REQUIRED issues and send another code review request.**

Cheers and Happy coding!👏👏👏

Code review by **Sarah Uryelah Silva Chamorro**

(Feel free to contact me on slack by the handle of  **sarah** if you have any additional question)

biodunJohnson commented:
The Clone New York Times Page

. Added the Navbar with fontawesome icon
. Added the middle contest with text and images
. Added sidebar image and text
. Added the footer

uryelah commented:
Good job fixing the last required changes.
About the message, next time make sure to add it at the top of the pull request. You can click at the three dots at the top right of the PR message, click edit at the menu and edit it whenever you need instead of making a new comment.

Keep an eye on those lowercase commits too, they should always be capitalized ;-)
You can read more on commit best practices[ here.](https://chris.beams.io/posts/git-commit/)

------
## Your project is approved! 🎊 🥅 💛 
### 
**You may close this Pull Request and merge the changes**

----

**Keep being an awesome dev and good luck with your next project!**

Cheers and Happy coding!👏👏👏

Code review by **Sarah Uryelah Silva Chamorro**

(Feel free to contact me on slack by the handle of  **sarah** if you have any additional question)
